package.json

```diff
@@ -20,6 +20,8 @@
     ]
   },
   "scripts": {
+    "prettify": "prettier --write 'src/**'",
```

Comment:
When running npm run prettify gives `[error] No parser could be inferred for file: src/.DS_Store`
and 
```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! design-tokens@0.0.1 prettify: `prettier --write 'src/**'`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the design-tokens@0.0.1 prettify script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```